First Backup Sync Implementation (#115)
nemtech/symbol-bootstrap: docs/start.md

  --args=args
      Add extra arguments to the docker-compose up command. Check out https://docs.docker.com/compose/reference/up.

  --fastSync

>       It downloads a backup with the mongo and rockdb databases to faster sync.

Reviewer: ```suggestion
      It downloads a backup with the mongo and rockdb databases for faster synchronization.
```
